### PR TITLE
ADD export to Google Sheets

### PR DESCRIPTION
**Pull request by pcooman**
I made an example version of the export job for PMI, the one that @aripollak helped me with. I think this could be of use for other projects.

### CODE REVIEW

File: export.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# This example workflow demonstrates how to create a parameterized Google Sheets Export job.
```
Comment (aripollak) [inline export.yml:1]: Maybe also add a comment about how the `export_to_google` task will be run once for every element in the `species` param?
Also would it be worth mentioning that the input parameters are the same as https://civis-python.readthedocs.io/en/stable/api_resources.html#civis.resources._resources.Imports.post and https://civis-python.readthedocs.io/en/stable/api_resources.html#civis.resources._resources.Imports.post_syncs ?

File: export.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                select * 
+                from pcooman.iris
+                where class = 'Iris-<% $.class %>'
+            destination:
+              google_worksheet:
+                spreadsheet: Workflow Demo -- Export to Google Sheet
+                worksheet: <% $.class %>
+            source:
+              path: >-
+                select *
```
Comment (aripollak) [inline export.yml]: I feel like we shouldn't need this and `advance_options.sql_query`. Does it work if you take one out, or leave one blank?

File: export.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  tasks:
+    export_to_google:
+      action: civis.import
+      with-items: 'class in <% $.species.split(",") %>' 
+      input:
+        name: 'Export Irises [<% $.class %>]'
+        sync_type: GdocExport
+        is_outbound: true
+        destination:
+          remote_host_id: 219
```
Comment (aripollak) [inline export.yml:32]: Replace these with placeholders since whoever uses this will probably need different ones?

**Review by aripollak — CHANGES_REQUESTED** [review]
Thanks for adding this, @pcooman!
@dianecarroll, do you want to look at this before it gets merged?

File: export.yml (hunk 1 above)
Comment (pcooman) [reply export.yml:1]: done!

File: export.yml (hunk 2 above)
Comment (pcooman) [reply export.yml]: no, removing either causes errors

File: export.yml (hunk 3 above)
Comment (pcooman) [reply export.yml:32]: done!

**Comment by pcooman** [comment]
@aripollak, @dianecarroll, I noticed that the YAML 'input' was not getting picked up as Platform Workflow parameters, unless I changed the overall workflow name to "workflow". I thought this name didn't matter, but apparently it does make a difference.

"workflow"
image

"anything_else"
image

**Review by aripollak — APPROVED** [review]
LGTM!

**Pull request merged**